backend/src/test/java/com/hjjang/backend/email/EmailRegexTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hjjang.backend.email;
+
+import static org.assertj.core.api.Assertions.*;
+
+import com.hjjang.backend.domain.email.exception.MailException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class EmailRegexTest {
+
+	private static final String REGEX = "^[a-zA-Z0-9]+@[a-zA-Z0-9]+\\.+a+c+\\.+k+r+$";
+	private static final String EMAIL_PARSE_REGEX = "[@.]";
+
+	@Test
+	void parseUniversity() {
+		//given
+		String email = "32174294@dankook.ac.kr";
+		//when
+		String university = List.of(email.split(EMAIL_PARSE_REGEX)).get(1);
+		//then
+		System.out.println("university = " + university);
+		assertThat(university).isEqualTo("dankook");
+	}
+
+	@Test
+	void checkInvalidEmail() {
+		//given
+		//when
+		List<String> falseEmails = new ArrayList<>();
+		falseEmails.add("32174294@naver.com");
+		falseEmails.add("32174294@gmail.com");
+		falseEmails.add("32174294_dankook.ac.kr");
+		falseEmails.add("@dankook.ac.kr");
+		falseEmails.add("32174294@ac.kr");
+		falseEmails.add("32174294@.ac.kr");
+		//then
+		falseEmails.forEach(email -> {
+			boolean match = Pattern.matches(REGEX, email);
+			System.out.println(email + " : " + match);
+			assertThat(match).isEqualTo(false);
+		});
+	}
+
+	@Test
+	void checkValidEmail() {
+		//given
+		//when
+		List<String> trueEmails = new ArrayList<>();
+		trueEmails.add("32174294@dankook.ac.kr");
+		trueEmails.add("32174294@anyang.ac.kr");
```

Comment:
```suggestion
		trueEmails.add("32174294@gs.anyang.ac.kr");
```
위에 처럼 안양대학교 메일 형태 정정입니다.

Reply:
뒤에서부터 확인하는 방법으로 정정하겠습니다.